pkg/option/config.go

```diff
@@ -1068,6 +1083,20 @@ var (
 	}
 )
 
+// IPv4NativeRoutingCIDR returns the native routing CIDR if configured
+func (c *DaemonConfig) IPv4NativeRoutingCIDR() *cidr.CIDR {
+	c.ConfigPatchMutex.RLock()
+	defer c.ConfigPatchMutex.RUnlock()
```

Comment:
consider removing the defer

Reply:
You really don't like these, don't you ;-)